frontend/packages/ceph-storage-plugin/src/components/modals/clone-pvc-modal/clone-pvc-modal.tsx

```diff
@@ -0,0 +1,174 @@
+    endpoint: PrometheusEndpoint.QUERY,
+    query: pvcUsedCapacityQuery,
+  });
+
+  const pvcUsedCapacityQueryResult: DataPoint[] = getInstantVectorStats(
+    response,
+    null,
+    humanizeBinaryBytes,
+  );
+  const pvcUsedCapacity: string = _.get(pvcUsedCapacityQueryResult[0], 'label')
+    ? _.get(pvcUsedCapacityQueryResult[0], 'label')
+    : 'No Data';
+
+  const submit = (event: React.FormEvent<EventTarget>) => {
+    event.preventDefault();
+
+    const pvcCloneObj = {
+      apiVersion: PersistentVolumeClaimModel.apiVersion,
+      kind: PersistentVolumeClaimModel.kind,
+      metadata: {
+        name: clonePVCName,
+        namespace: resource.metadata.namespace,
+      },
+      spec: {
+        storageClassName: resource.spec.storageClassName,
+        dataSource: {
+          name: pvcName,
+          kind: PersistentVolumeClaimModel.kind,
+          apiGroup: '',
+        },
+        accessModes: resource.spec.accessModes,
+        resources: {
+          requests: {
+            storage: resource.spec.resources.requests.storage,
+          },
+        },
+      },
+    };
+
+    handlePromise(k8sCreate(PersistentVolumeClaimModel, pvcCloneObj))
+      .then(() => {
+        close();
+      })
+      .catch((err) => {
+        setSubmitDisabled(true);
+        throw err;
+      });
+  };
+
+  return (
+    <Form onSubmit={submit}>
+      <div className="modal-content modal-content--no-inner-scroll">
+        <ModalTitle>Clone</ModalTitle>
+        <ModalBody>
+          <FormGroup label="Name" isRequired fieldId="ceph-clone-pvc-modal__name">
+            <TextInput
+              isRequired
+              type="text"
+              id="ceph-clone-pvc-modal__name"
+              name="ceph-clone-pvc-modal__name"
+              value={clonePVCName}
+              onChange={(value) => setClonePVCName(value)}
+            />
+          </FormGroup>
+          <div className="ceph-clone-pvc-modal__details">
+            <p className="text-muted ceph-clone-pvc-modal__details-label">PVC Details</p>
+            <div className="ceph-clone-pvc-modal__details-section">
+              <div>
+                <div>
+                  <p className="ceph-clone-pvc-modal__details-label">Namespace</p>
+                  <p>
+                    <ResourceIcon kind="Namespace" />
+                    {resource.metadata.namespace}
+                  </p>
+                </div>
+                <div>
+                  <p className="ceph-clone-pvc-modal__details-label">Storage Class</p>
+                  <p>
+                    <ResourceIcon kind="StorageClass" />
+                    {resource.spec.storageClassName}
+                  </p>
+                </div>
+              </div>
+              <div>
+                <div>
+                  <p className="ceph-clone-pvc-modal__details-label">Requested Capacity</p>
+                  <p>
+                    {
+                      humanizeBinaryBytes(
```

Comment:
Why not just show the original value as is?

If the original value was using decimal units, this will show a decimal instead of the original round value.